web-ui/src/components/member-directory/MemberModal.jsx

```diff
@@ -59,31 +61,118 @@ const MemberModal = ({ member, open, onSave, onClose }) => {
     });
   };
 
-  const validateRequiredInputs = useCallback(() => {
-      return (editedMember.firstName?.length > 0 && editedMember.lastName?.length > 0
-         && editedMember.workEmail?.length > 0 && editedMember.title?.length > 0 && editedMember.location?.length >0 && editedMember.employeeId?.length > 0 && startDate )
-  }, [editedMember, startDate])
+  const validateInputs = useCallback(() => {
+    let regName = /^[a-zA-Z]+$/;
+    if (
+      !regName.test(
+        editedMember.firstName ||
+          !regName.test(editedMember.lastName) ||
+          !regName.test(editedMember.middleName)
+      )
+    ) {
+      dispatch({
+        type: UPDATE_TOAST,
+        payload: {
+          severity: "error",
+          toast:
+            "Please enter a valid name without numbers or special characters",
+        },
+      });
+      return false;
+    }
+    let regEmail = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
```

Comment:
for your consideration:
https://emailregex.com/

Reply:
Edited per email regex suggestion.